src/components/Menu/MenuItem/MenuItem.module.scss

```diff
@@ -37,6 +37,31 @@
   color: var(--icon-color);
 }
 
+.subMenuIconWrapper .divider {
+  height: 19px;
+}
+
+.subMenuIconWrapper .subMenuSplitButtonIcon {
+  width: 34px;
+  height: 30px;
+  --primary-background-hover-color: rgba(103, 104, 121, 0.1);
```

Comment:
Why this override is needed? Temp fix for upcoming hover color change?
If so, then corresponding TODO comment would be appreciated

Reply:
removing, this is just to check how the menu would look like if we will have the hover color that Tal added